### DISCUSSION TITLE
doris 1.2.0版本的jdbc外部表扩展文章消失了？

### DISCUSSION THREAD

**Discussion by magicxf** [apache/doris#16292] [A - General / Q&A] [answered]
https://doris.apache.org/zh-CN/docs/dev/releasenotes/release-1.2.0#5-jdbc-%E5%A4%96%E9%83%A8%E8%A1%A8
Release 1.2.0更新文档中提到了jdbc外部表
![image](https://user-images.githubusercontent.com/28557507/215731394-85e368ba-7794-422c-8733-b1d34353b8bc.png)

其中的jdbc外部表找不到了：
https://doris.apache.org/zh-CN/docs/dev/ecosystem/external-table/jdbc-of-doris/

**Comment by gnehil** [comment, marked as answer]
https://doris.apache.org/zh-CN/docs/dev/lakehouse/external-table/jdbc
外表的文档移动到这里了
